QRScanner/QRScannerView.swift

```diff
@@ -80,7 +80,8 @@ public class QRScannerView: UIView {
         assert(Thread.isMainThread)
         
         guard let videoDevice = AVCaptureDevice.default(for: .video),
-            videoDevice.hasTorch, videoDevice.isTorchAvailable else {
+            videoDevice.hasTorch, videoDevice.isTorchAvailable,
+            (metadataOutputEnable || videoDataOutputEnable) else {
```

Comment:
Before `startRunning` (= start QR scanning)

  - `metadataOutputEnable`: false
  - `videoDataOutputEnable`: false

In `startRunning`

  - `metadataOutputEnable`: true
  - `videoDataOutputEnable`: true

After `metadataOutput`

  - `metadataOutputEnable`: false
  - `videoDataOutputEnable`: true

After `captureOutput` (= end QR scanning)

  - `metadataOutputEnable`: false
  - `videoDataOutputEnable`: false